bin/server.js

```diff
@@ -73,7 +74,8 @@ var launch = function(profile) {
       context: {
         validator:      validator,
         Client:         Client,
-        azureAccounts:  JSON.parse(cfg.get('auth:azureAccounts'))
+        azureAccounts:  JSON.parse(cfg.get('auth:azureAccounts')),
+        sts:            new AWS.STS(cfg.get('aws'))
```

Comment:
Is this only for a single region?  Does it matter that we're going to have instances in other regions as well?